### PR TITLE
py-[semver|upt-pypi]: update to latest versions

### PR DESCRIPTION
**Pull request by reneeotten**
#### Description
- py-semver: update to 2.10.1, fix dependency type
- py-upt-pypi: update to 0.5.1 

###### Tested on
macOS 10.14.6 18G4032
Xcode 10.3 10G8

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
Is there a reason you have put these two updates in on PR ? Does one update require the other ?

**Comment by reneeotten** [comment]
> Is there a reason you have put these two updates in on PR ? Does one update require the other ?

`py-semver` is used by one of the `upt`-related ports; the main reason I put them in one PR is that it's a very simple version bump of two ports from the same maintainer. So if Karan sees the PR he can just look at both ports at the same time - easier for both of us.

File: python/py-semver/Portfile [hunk 1]
```diff
@@ -17,23 +17,24 @@ long_description    ${description}
...
+checksums           sha256  b08a84f604ef579e474ce448672a05c8d50d1ee0b24cee9fb58a12b260e4d0dc \
+                    rmd160  8e2e82dff987cc8d97e2769e44546d8d009148b0 \
+                    size    21661
 
 python.versions     37 38
 
 if {${name} ne ${subport}} {
-    depends_build-append \
+    depends_lib-append \
                     port:py${python.version}-setuptools
```
Comment (cjones051073) [inline python/py-semver/Portfile:28]: Why this change ? Setuptools is commonly a build dep. only. Does this port really need it at runtime too ?

Comment (Korusuke) [reply python/py-semver/Portfile:28]: Starting from 2.9.1 semver introduces a console script and hence setuptools is required at runtime.
